docs/ref/settings.txt

```diff
@@ -1836,6 +1836,20 @@ Here's a sample settings file::
         ('en', _('English')),
     ]
 
+.. setting:: LANGUAGES_BIDI
+
+``LANGUAGES_BIDI``
+------------------
+
+Default: A list of all languages in the :setting:`LANGUAGES` setting that are
+written right-to-left.
+
+The list should contain :term:`language code<language code>` strings for
+languages that are written right-to-left.
+
+Generally, the default value should suffice. Only set this setting if you want
```

Comment:
IMO, we can omit this sentence `Generally, the default value should suffice.`.

Reply:
This follows the same wording as for the `LANGUAGES` setting.

Reply:
:thinking: ok, for me it sounds redundant with _"Only set this ..."_ :man_shrugging: